### PR TITLE
Fixed unnecessary GDAL dependency.

### LINKED ISSUE #75: Serializer still depends on GDAL?
I encountered a strange issue. On my development machine, everything worked fine. When I tried to run my project on another computer, I suddenly got errors like the following when saving a PointField:

``` python
>>> self.pointField = {'type': 'Point', 'coordinates': [lon, lat]}
>>> self.save()
[2016-28-23 20:28:53] ERROR [django.contrib.gis:45] Error creating geometry from value '{
  "type":"Point",
  "coordinates":[
    7.09,
    50.72
  ]
}' (Initializing geometry from JSON input requires GDAL.)
```

After performing a `sudo apt-get install gdal-bin` (which also installed a whole bunch of other geospatial libraries), the errors were gone. But the whole point of using django-geojson is that I don't need to install geospatial libraries just to store some Points and have nice Leaflet widgets.

On both computers, Django runs in the same freshly installed Python3-virtualenv:

``` bash
$ pip freeze
Django==1.9.2
PyYAML==3.11
crispy-forms-foundation==0.5.3
django-crispy-forms==1.6.0
django-geojson==2.9.0
django-leaflet==0.18.0
jsonfield==1.0.3
mysqlclient==1.3.7
pytz==2015.7
six==1.10.0
xlwt==1.0.0
```

### PR DESCRIPTION
**Pull request by coredumperror**
This PR fixes #75, as far as I can tell. At least, I can get through a `manage.py makemigrations` now while using `GeoJSONLayerView` in one of my files. There may be other places where this check for `ImproperlyConfigured` needs to be added.

### CODE REVIEW

File: djgeojson/views.py [hunk 1]
```diff
@@ -149,7 +150,8 @@ def get_queryset(self):
         # Won't trim point geometries to a boundary
         model_field = qs.model._meta.get_field(self.geometry_field)
         self.trim_to_boundary = (self.trim_to_boundary and
-                                 not isinstance(model_field, PointField))
+                                 not isinstance(model_field, PointField) and
+                                 Intersection)
```
Comment (leplatrem) [inline djgeojson/views.py]: `Intersection is not None` would be easier to read I think

Comment (coredumperror) [reply djgeojson/views.py]: Good point. Fix incoming.

**Comment by coredumperror** [comment]
The failing tests are all for Django 1.8, which has been desupported for more than 6 months. Every other test suite passes. 

I could change the patch to not affect the Django 1.8 code path, but then the two code paths would have different behavior based on whether GDAL is installed or not, even though the Django 1.8 code (apparently) doesn't use it. 

So I'm not sure what the right solution is.

**Comment by leplatrem** [comment]
The devs at makinacorpus should confirm if they're ok with that breaking change (ie. dropping 1.8)

Otherwise, at least drop them from Travis

**Comment by leplatrem** [comment]
@gutard @Gagaro @LePetitTim ping?

**Comment by gutard** [comment]
> The devs at makinacorpus should confirm if they're ok with that breaking change (ie. dropping 1.8)

Geotrek now requires Django 1.11 so it's OK to drop Django 1.8 support.

**Comment by Gagaro** [comment]
I'm ok with dropping support for 1.8 as well. All the travis test are failing because of travis though, I'll take care of fixing everything and updating the travis conf.

**Comment by Gagaro** [comment]
Released in 2.12.0. Thanks @coredumperror :+1: .

**Pull request merged**